utility/src/main/java/org/oppia/util/extensions/LiveDataExt.kt

```diff
@@ -0,0 +1,14 @@
+package org.oppia.util.extensions
+
+import androidx.lifecycle.LifecycleOwner
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.Observer
+
+fun <T> LiveData<T>.observeOnce(lifecycleOwner: LifecycleOwner, observer: Observer<T>) {
```

Comment:
Please add KDoc explaining what this does & how to use it.

Reply:
Also, thanks for adding this. :D

Reply:
Updated.